db/seeds.rb

```diff
@@ -0,0 +1,67 @@
+require 'bundler/setup'
+require "pg"
+require "active_record"
+require "pry"
+
+require_relative "../models/tenant"
+require_relative "../models/apartment"
+
+require_relative "../connection"
```

Comment:
I recommend connecting to the database prior to requiring the models, because they depend on the database.